scss/partials/_columns.scss

```diff
@@ -103,3 +103,33 @@
     }
   }
 }
+
+.l-columns {
+  @include breakpoint(tablet) {
+    &--right-sidebar {
+      @include grid(11.4rem);
+      grid-template-columns: 1.82fr 1fr;
+    }
+  }   
+
+  &__heading {
+    margin-bottom: 7.5rem;
+  }
+
+  &__section-separator {
+    background: none;
+    border-top: 1px solid $cool-neutral-6;
+    margin: 5rem 0;
+  }
+
+  &__subheading {
+    color: $primary-blue;
+    display: block;
+    margin-bottom: 2.5rem;
+  }
+
+  &__tip {
+    display: block;
+    margin-top: 2.6rem;
+  }
```

Comment:
I might break these out into something more like an `l-sidebar` block later, as that feels maybe more appropriate.